layers/+lang/scala/packages.el

```diff
@@ -81,6 +86,20 @@
         (add-to-list 'completion-ignored-extensions ext)))
     :config
     (progn
+
+      ;; Ensure only one of metals and ensime is loaded
+      (unless (spacemacs//scala-backend-ensime)
+        (progn
+          (fmakunbound 'ensime)
+          (remove-hook 'after-change-functions 'ensime-after-change-function)
+          (remove-hook 'window-configuration-change-hook 'ensime-show-left-margin-hook)))
+
+      ;; Auto start metals
+      (if (spacemacs//scala-backend-metals)
+          (add-hook 'scala-mode-hook #'lsp)
+        (with-eval-after-load 'lsp
```

Comment:
this is not needed - lsp layer is setting lsp-prefer-make to nil

Reply:
tweaked